lib/iris/__init__.py

```diff
@@ -187,9 +187,15 @@ def __init__(self, cell_datetime_objects=True, netcdf_promote=True,
         unlimited.  The current default is that the leading dimension is
         unlimited unless otherwise specified.
 
-        The option `clip_latitudes` controls whether the
-        :meth:`iris.coords.Coord.guess_bounds()` method limits the
-        guessed bounds to [-90, 90] for latitudes.
+        .. deprecated:: 2.0.0
+
+            The option `clip_latitudes` is deprecated and will be removed in a
+            future release and the deprecated code paths this option used to
+            toggle have been removed.
```

Comment:
This isn't true; it's still possible to toggle the different code paths, and neither of them were deprecated anyway.